tket/src/Circuit/CircUtils.cpp

```diff
@@ -618,6 +619,40 @@ static Circuit CnU1(unsigned n_controls, Expr lambda) {
   }
 }
 
+/**
+ * @brief Returns the controlled version of a ConjugationBox
+ * The returned circuit is box free
+ * @param op assumes to be ConjugationBox
+ * @param n_controls
+ * @param args qubits where the box is original placed
```

Comment:
There's an assumption that the args are all singly-indexed qubits in the same register. Looks like this is always the case when this function is called, but worth documenting here (and perhaps even asserting)?